AUTHORS.rst

```diff
@@ -16,3 +16,5 @@ The following organizations or individuals have contributed to ScanCode:
 - Tushar Mittal @techytushar
 - Martin Petkov @MartinPetkov
 - Mrinal Paliwal @mriiinal
+- Mankaran Singh @MankaranSingh
+- Gaurang Rao @Gaupeng
```

Comment:
Is this a joint development? Why do you add @MankaranSingh name?

Reply:
Whoops, big oopsie on my side. WIll fix it.